contributing/development/compiling/compiling_for_macos.rst

```diff
@@ -152,6 +152,14 @@ with the following commands (assuming a universal build, otherwise replace the
     cp bin/godot.macos.template_debug.universal macos_template.app/Contents/MacOS/godot_macos_debug.universal
     chmod +x macos_template.app/Contents/MacOS/godot_macos*
 
+For a binary compiled with double precision, (for example `scons platform=macos arch=arm64 precision=double`) the following will work:
```

Comment:
For when you reopen a new version, you need to have two colons here, like so:
```suggestion
For a binary compiled with double precision, (for example `scons platform=macos arch=arm64 precision=double`) the following will work::
```
See the one above